Projects/SelectTier/Sources/Views/EditDeductible.swift

```diff
@@ -0,0 +1,74 @@
+import SwiftUI
+import hCore
+import hCoreUI
+
+struct EditDeductible: View {
+    @State var selectedDeductible: String?
+    var vm: SelectTierViewModel
+    @EnvironmentObject var selectTierNavigationVm: SelectTierNavigationViewModel
+
+    init(
+        vm: SelectTierViewModel
+    ) {
+        self.vm = vm
```

Comment:
Also, here you can set selectedDeductible like
self._ selectedDeductible = .init...